app/views/federations/show.json.jbuilder

```diff
@@ -0,0 +1,12 @@
+
+@federated_results[:more_results_label] = t('global.actions.view-more-at') + @foederati_provider.display_name
```

Comment:
agreed, this is much nicer than the copy paste job I did.
`json.img { src: result[:thumbnail] }` 
needs to be 
` json.img { json.src result[:thumbnail] }`